src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -91,8 +87,8 @@
                      assoc
                      :suggested-routes          suggested-routes-data
                      :route                     chosen-route
-                     :from-values-by-chain      from-network-values-for-ui
-                     :to-values-by-chain        to-network-values-for-ui
+                     :from-values-by-chain      from-network-amounts-by-chain
+                     :to-values-by-chain        to-network-amounts-by-chain
```

Comment:
Here's where we were storing the UI strings inside the re-frame state. Now we'll store the original data and do the conversions inside the UI components.